cmake/pcl_options.cmake

```diff
@@ -1,7 +1,10 @@
 # Options for building PCL.
 
 # Build shared libraries by default.
-option(PCL_SHARED_LIBS "Build shared libraries." ON)
+include(CMakeDependentOption)
```

Comment:
One final remark, since CMakeDependOption, is useful project wide, it's better to invoke the `include` statement at global scope.

Somewhere before [here](https://github.com/PointCloudLibrary/pcl2/blob/master/CMakeLists.txt#L12) which is where all the other includes start taking place.

Reply:
@SergioRAgostinho 

Thank you for the review. It has been changed.